src/scripts/components/Play.js

```diff
@@ -35,22 +35,43 @@ type State = {
 }
 
 const Wrapper = styled.div`
-  width: 100%;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  flex: 0 0 100%;
+  margin: 0 auto;
```

Comment:
@jrgarou I just want to confirm, does this work well for both the `/play` and `/embed` routes?

Reply:
@bent0b0x now it is working https://github.com/Paratii-Video/paratii-portal/pull/275/commits/6fb93d194ab2ae3e04fa599fe7cabdf40b3b5d6c